lib/Reconciler.lua

```diff
@@ -408,12 +394,20 @@ function Reconciler._reconcilePrimitiveProps(fromElement, toElement, rbx)
 			local oldValue = fromElement.props[key]
 
 			if oldValue ~= newValue then
-				Reconciler._setRbxProp(rbx, key, newValue)
+				Reconciler._setRbxProp(rbx, key, newValue, toElement)
 			end
 		end
 	end
 end
 
+--[[
+	Used in _setRbxProp to avoid creating a new closure for every property set.
+]]
+
+local function _rawSet(rbx, key, value)
```

Comment:
I understand the allusion to `rawset` with this function name, but this version isn't very raw, is it? Would there be any problem with just naming this function `set`?

As a small style nitpick, you should remove the blank line after the documentation comment to group it visually with this function.